pkg/lint/lint.go

```diff
@@ -39,11 +42,36 @@ var (
 	ErrNotInstaller = fmt.Errorf("object is not an installer")
 )
 
+type Severity string
+
+const (
+	SeverityError   Severity = "error"
+	SeverityWarning Severity = "warning"
+	SeverityInfo    Severity = "info"
+)
+
 // Output holds the outcome of a lint pass on top of a Installer struct.
 type Output struct {
-	Field   string `json:"field,omitempty"`
-	Type    string `json:"type,omitempty"`
-	Message string `json:"message,omitempty"`
+	Field    string          `json:"field,omitempty"`
```

Comment:
```suggestion
	Field    string          `json:"field"`
```

Could we have a lint check without define the spec/field to be checked?